gazebo_ros/test/CMakeLists.txt

```diff
@@ -22,8 +23,18 @@ foreach(plugin ${plugins})
   )
 endforeach()
 
+# Worlds
+set(worlds
+  ros_world_plugin
+)
+
+foreach(world ${worlds})
+  configure_file(worlds/${world}.world ${world}.world COPYONLY)
+endforeach()
```

Comment:
May be simpler to just install the worlds directory unless we plan on having non-world files in there

Reply:
You mean just configuring the whole directory (i.e. copying to the build folder) or actually installing? I agree with configuring, but I don't think it's a good idea to install them since they're specific to tests; otherwise we'd need to install the plugins too for example.

Reply:
Just copying the whole worlds directory to build: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/776/commits/10be195185bca8015c02a391f628cfbabf944216